features/Websocket.feature

```diff
@@ -0,0 +1,10 @@
+Feature: WebSocket
+
+  # WebSocket ===========================================================
+
+  Scenario: Check reponse message from Kuzzle with a specific payload
+    Given I open a new local websocket connection
+    When I send the message '{"p":"1"}' to Kuzzle through websocket
+    Then I should receive a response message from Kuzzle through websocket matching:
+      |  p  |  '1'  |
+    Then I terminate the websocket connection
```

Comment:
This should be a hook on a tag (like the one on `@securities`) because if the step line 8 fail then the step line 10 will not be executed and the websocket connexion will not be terminated